src/makeRobot.js

```diff
@@ -38,7 +38,74 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const obj = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      const vers = this.version;
+      const wheel = this.wheels;
+      const robot = this.name;
```

Comment:
```suggestion
      const wheels = this.wheels;
      const robotName = this.name;
```
Variable robot sounds like the whole robot object, not like a string with robot's name. So it can be confusing